bindings/python/python_constants.cpp

```diff
@@ -800,279 +800,5 @@ void addLabels(PyObject * dict)
-    ADD_CONST(BSOFT_PARTICLE_PRJ_CC);
-    ADD_CONST(BSOFT_PARTICLE_CMP_CC);
-    ADD_CONST(BSOFT_PARTICLE_RFACTORAB);
-    ADD_CONST(BSOFT_PARTICLE_COVERAGE);
-    ADD_CONST(BSOFT_PARTICLE_BOX_SIZE);
-    ADD_CONST(BSOFT_PARTICLE_BOX_SIZE_X);
-    ADD_CONST(BSOFT_PARTICLE_BOX_SIZE_Y);
-    ADD_CONST(BSOFT_PARTICLE_BOX_SIZE_Z);
-    ADD_CONST(BSOFT_PARTICLE_BOX_RADIUS);
-    ADD_CONST(BSOFT_PARTICLE_BOX_RADIUS_X);
-    ADD_CONST(BSOFT_PARTICLE_BOX_RADIUS_Y);
-    ADD_CONST(BSOFT_PARTICLE_BOX_RADIUS_Z);
-    ADD_CONST(BSOFT_PARTICLE_BAD);
-    ADD_CONST(BSOFT_PARTICLE_BAD_RADIUS);
-    ADD_CONST(BSOFT_PARTICLE_BAD_X);
-    ADD_CONST(BSOFT_PARTICLE_BAD_Y);
-    ADD_CONST(BSOFT_PARTICLE_BAD_Z);
-    ADD_CONST(BSOFT_FILAMENT);
-    ADD_CONST(BSOFT_FILAMENT_FILE);
-    ADD_CONST(BSOFT_FILAMENT_ID);
-    ADD_CONST(BSOFT_FILAMENT_NODE);
-    ADD_CONST(BSOFT_FILAMENT_NODE_ID);
-    ADD_CONST(BSOFT_FILAMENT_NODE_X);
-    ADD_CONST(BSOFT_FILAMENT_NODE_Y);
-    ADD_CONST(BSOFT_FILAMENT_NODE_Z);
-    ADD_CONST(BSOFT_FILAMENT_WIDTH);
-    ADD_CONST(BSOFT_FILNODE_RADIUS);
-    ADD_CONST(BSOFT_ORIENT_ID);
-    ADD_CONST(BSOFT_ORIENT_ORIGIN_X);
-    ADD_CONST(BSOFT_ORIENT_ORIGIN_Y);
-    ADD_CONST(BSOFT_ORIENT_ORIGIN_Z);
-    ADD_CONST(BSOFT_ORIENT_VIEW_X);
-    ADD_CONST(BSOFT_ORIENT_VIEW_Y);
-    ADD_CONST(BSOFT_ORIENT_VIEW_Z);
-    ADD_CONST(BSOFT_ORIENT_VIEW_ANGLE);
-    ADD_CONST(BSOFT_ORIENT_FOM);
-    ADD_CONST(BSOFT_ORIENT_SELECT);
-    ADD_CONST(BSOFT_MARKER);
-    ADD_CONST(BSOFT_MARKER_RADIUS);
-    ADD_CONST(BSOFT_MARKER_ID);
-    ADD_CONST(BSOFT_MARKER_X);
-    ADD_CONST(BSOFT_MARKER_Y);
-    ADD_CONST(BSOFT_MARKER_Z);
-    ADD_CONST(BSOFT_MARKER_ERROR_X);
-    ADD_CONST(BSOFT_MARKER_ERROR_Y);
-    ADD_CONST(BSOFT_MARKER_ERROR_Z);
-    ADD_CONST(BSOFT_MARKER_IMAGE);
-    ADD_CONST(BSOFT_MARKER_RESIDUAL);
-    ADD_CONST(BSOFT_MARKER_FOM);
-    ADD_CONST(BSOFT_MARKER_SELECT);
-    ADD_CONST(BSOFT_REFLEX);
-    ADD_CONST(BSOFT_REFLEX_RADIUS);
-    ADD_CONST(BSOFT_REFLEX_X);
-    ADD_CONST(BSOFT_REFLEX_Y);
-    ADD_CONST(BSOFT_REFLEX_Z);
-    ADD_CONST(BSOFT_REFLEX_H);
-    ADD_CONST(BSOFT_REFLEX_K);
-    ADD_CONST(BSOFT_REFLEX_L);
-    ADD_CONST(BSOFT_REFLEX_AMP);
-    ADD_CONST(BSOFT_REFLEX_SIGAMP);
-    ADD_CONST(BSOFT_REFLEX_PHI);
-    ADD_CONST(BSOFT_REFLEX_SIGPHI);
-    ADD_CONST(BSOFT_REFLEX_FOM);
-    ADD_CONST(BSOFT_REFLEX_STATUS);
-    ADD_CONST(BSOFT_LAYERLINE);
-    ADD_CONST(BSOFT_LAYERLINE_NUMBER);
-    ADD_CONST(BSOFT_LAYERLINE_ORDER);
-    ADD_CONST(BSOFT_LAYERLINE_DISTANCE);
-    ADD_CONST(BSOFT_LAYERLINE_FREQ);
-    ADD_CONST(BSOFT_LAYERLINE_AMP);
-    ADD_CONST(BSOFT_LAYERLINE_FOM);
-    ADD_CONST(BSOFT_LAYERLINE_SELECT);
-
-    ADD_CONST(BSOFT_SYMMETRY_INT_TABLES_NUMBER);
-    ADD_CONST(BSOFT_SYMMETRY_SPACE_GROUP_NAME_H_M);
-    ADD_CONST(BSOFT_SYMMETRY_CELL_SETTING);
-    ADD_CONST(BSOFT_SYMMETRY_EQUIV_ID);
-    ADD_CONST(BSOFT_SYMMETRY_EQUIV_POS_AS_XYZ);
-
-    // BUFFER labels
-    ADD_CONST(BUFFER_LABELS_START);
```

Comment:
I understand that since [the "Automatic mapping of..." PR](https://github.com/I2PC/xmippCore/pull/7), these buffer labels should not be added anymore.

Reply:
I mean, not only the first one, but also the rest

Reply:
We need the buffer labels in the C++ code, but them will be mapped automatically, so I don't see the need to add it to the Python binding...but I'm also fine with adding them if someone thinks could be useful.